ztp/source-crs/ImageRegistryConfig.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: imageregistry.operator.openshift.io/v1
+kind: Config
+metadata:
+  name: cluster
+  annotations:
+    # The registry depends on backend storage being
+    # configured. StoragePVC defaults to wave 10, so this CR defaults
+    # to wave 11 to satisfy that dependency.
+    ran.openshift.io/ztp-deploy-wave: "11"
+spec:
+  logLevel: Normal
+  managementState: Managed
+  observedConfig: null
+  # These fields can be set by the user as desired
+  #operatorLogLevel: Normal
+  #proxy: {}
+  replicas: 1
+  requests:
+    read:
+      maxWaitInQueue: 0s
+    write:
+      maxWaitInQueue: 0s
+  rolloutStrategy: Recreate
+  # User needs to specify proper storage
+  storage: {}
+  #  pvc:
+  #    claim: ''
```

Maybe use a template-like example for the parts we expect users to fill in?
```suggestion
  storage:
    pvc:
      claim: '$pvcname'
```